tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Translation.cpp

```diff
@@ -93,16 +103,52 @@ void test_translation() {
       radius += square(distance_to_center[i]);
     }
     radius = sqrt(radius);
+    double inspiral_radius = magnitude(point_xi);
+    double radial_falloff_factor =
+        (outer_radius - inspiral_radius) / (outer_radius - inner_radius);
     std::array<double, Dim> frame_vel{};
     std::array<double, Dim> radial_frame_vel{};
+    std::array<double, Dim> inspiral_frame_vel{};
+    std::array<double, Dim> far_inspiral_frame_vel{0.};
     for (size_t i = 0; i < Dim; i++) {
       gsl::at(frame_vel, i) = f_of_t->func_and_deriv(t)[1][i];
       gsl::at(radial_frame_vel, i) =
           f_of_t->func_and_deriv(t)[1][i] * gaussian(radius);
+      if (inspiral_radius <= inner_radius) {
+        gsl::at(inspiral_frame_vel, i) = f_of_t->func_and_deriv(t)[1][i];
+      } else if (inspiral_radius > inner_radius &&
```

Comment:
I don't think this needs to be an `else if`, just an `else` because the point distribution for the inner radius is always less than the outer radius and you just checked if the point was inside the inner radius (same below)